docs/intro.rst

```diff
@@ -56,24 +58,10 @@ the Camus's jar is under ``/usr/share/java/camus/``.
 Also, you need to have access to a Hadoop Cluster. For installation and deployment of a single node
 Hadoop in pseudo-distributed mode, see this
 `guide <http://www.cloudera.com/content/cloudera/en/documentation/core/latest/topics/cdh_qs_cdh5_pseudo.html>`_.
+Finally, you should have Kafka and the Schema Registry running.
```

Comment:
Can we link back to our quick start for getting Kafka and Schema Registry started?